### ISSUE TITLE
Potentially confusing scaling of transition count matrix

### ISSUE THREAD

**Issue by synapticarbors** [msmbuilder/msmbuilder#436]
In digging through the source code, I noticed something that could be potentially confusing to users. In defining the graph in `strongly_connected_subgraph` there is a comparison done between the counts and the `weight` argument to the method:

https://github.com/msmbuilder/msmbuilder/blob/master/msmbuilder/msm/core.py#L203

This is clearly documented in the docstring. However `_transition_counts()` doesn't actually return the observed transitions, but rather the observed transitions scaled by the lag time:

https://github.com/msmbuilder/msmbuilder/blob/master/msmbuilder/msm/core.py#L330

I'm not sure what the rationale is for this, but what it means is that you can have fractional counts that fall below the default weight cut-off of `1` and thus not contribute an edge in the graph when determining the largest ergodic set. So for example:

```
T, mapping = _transition_counts([[0,1,0,1,0,1,0,1]], lag_time=3)
```

returns:

```
[[ 0.          1.        ]
 [ 0.66666667  0.        ]]
```

rather than what I would expect from the count matrix:

```
[[ 0.  3.]
 [ 2.  0.]]
```

Passing the fractional count matrix to `_strongly_connected_subgraph` with the default `weight=1` results in 2 strongly connected components rather than 1 if you actually had `T * lag_time` as the count matrix.

Maybe there is a good modeling reason for this (?), but it's not clear that this would happen from the doc strings and it seems unexpected in general.

**Comment by kyleabeauchamp** [comment]
I believe the scaling was introduced to correct the error estimates on rate and transition matrices when using a sliding window (e.g. non-independent counts).

**Comment by mpharrigan** [comment]
I think you're right, but this is confusing re: cutoff

**Comment by rmcgibbo** [comment]
> I believe the scaling was introduced to correct the error estimates on rate and transition matrices when using a sliding window (e.g. non-independent counts).

Yes, this hasn't been described in the literature yet, but it's something I'm working on. I think I can demonstrate quantitatively that the scaling gives substantially improved error estimators, at least for simple systems where we can generate `N` independent collections of trajectories for the same system. Note that scaling doesn't change the MLE.

> Passing the fractional count matrix to _strongly_connected_subgraph with the default weight=1 results in 2 strongly connected components rather than 1 if you actually had T * lag_time as the count matrix.

Maybe we should apply the weight cutoff before the scaling? Or we could just improve the docstring. I'm not really sure which is preferable. The effects of ergodic trimming (and the proper way to tune it, etc) is very tricky to analyze analytically.

**Comment by rmcgibbo** [comment]
Fixed by #439.
